resources/deu.ini

```diff
@@ -309,3 +309,16 @@ buy.notforsale=Dieses Grundstück wird nicht verkauft
 buy.confirm=Möchtest du wirklich das Grundstück {%0} für ${%1} kaufen? Um es zu bestätigen, benutze /p buy confirm
 buy.success=Du hast das Grundstück {%0} erfolgreich für ${%1} erworben
 buy.sold=Der Spieler {%0} hat dein Grundstück ({%1}) für ${%2} gekauft
+
+; The /p merge Subcommand
+merge.name=merge
+merge.desc=Füge ein Grundstück in der angegebenen Richtung zusammen
+merge.usage=/p merge [nord|süd|ost|west]
+merge.confirmarg=Bist du dir sicher das du das Plot {%0} {%1} zusammenfügen möchtest? Die änderung ist für immer. Wenn ja, verwende /p merge {%2}
```

Comment:
- and we're using "Grundstück" instead of "Plot"
- "änderung" has to "Änderung"
- "sicher das" has to be "sicher, dass"